playground/images/index.ts

```diff
@@ -0,0 +1 @@
+export {default as FileUploadSpot} from './File-Upload-Spot.png';
```

Comment:
🔥 ?

Reply:
Thanks for catching, meant to 🔥 this! That was left over from when I'd saved the empty state image locally before figuring out I could reference images attached to GitHub comments 👍.